ext/xmlreader/php_xmlreader.stub.php

```diff
@@ -4,6 +4,34 @@
 
 class XMLReader
 {
+    public int $attributeCount;
```

Comment:
You suggested previously to add `@readonly` to such cases. I think it's a good idea, so we know what to adjust if we get readonly or accessor support in the future.

Reply:
Yeah, I still like the idea, so I'll do a followup PR to mark all the readonly properties. :)

P.S.: Thanks for your commit, I missed the warning in the read handler.